Bindings/OpenSimHeaders_simulation.h

```diff
@@ -138,5 +138,12 @@
 #include <OpenSim/Simulation/StatesTrajectory.h>
 #include <OpenSim/Simulation/StatesTrajectoryReporter.h>
 
+#include <OpenSim/Actuators/osimActuatorsDLL.h>
+#include <OpenSim/Actuators/MuscleFixedWidthPennationModel.h>
```

Comment:
Is it necessary to include these Actuator headers in OpenSimHeaders_simulation.h? Currently, each OpenSimHeaders_[blah].h file strictly includes headers from OpenSim/[blah]. Why is it now necessary to deviate from this pattern?